fsspec/implementations/cached.py

```diff
@@ -202,6 +219,7 @@ def clear_cache(self):
         """
         rmtree(self.storage[-1])
         self.load_cache()
+        self._cache_size = None
```

Comment:
Should be 0 ?

Reply:
Unfortunately not, as the `self.load_cache()` call in the line above creates the cache directory which has OS-specific non-zero size.